### ISSUE TITLE
Postgres + scram-sha-256 authentication

### ISSUE THREAD

**Issue by GeoJGH** [HeidiSQL/HeidiSQL#831]
## Steps to reproduce this issue

1. Get a Postgres server using scram-sha-256 authentication
2. Try to connect using HeidiSQL

## Current behavior

It cannot connect. Error message is "authentication method 10 not supported"

## Expected behavior

Successful connection

## Possible solution

Replace libpq.dll with one from Postgres 10 install (it does not work with one from a Postgres 12 install)

## Environment

* **HeidiSQL version:** 
Version 10.3.0.5771 (64 Bit)
  

* **Database system and version:** 
  
Server was Postgres 10.3 and 12.1
This authentication method was introduced in v10

* **Operating system:** 
  
Windows 10

**Comment by ansgarbecker** [comment]
Related: https://www.heidisql.com/forum.php?t=26894

**Comment by maxs-s** [comment]
I have managed to connect to PostgreSQL database via HeidiSQL on windows. 
Steps that worked for me:
1. Copy the libpq.dll from postgresql-12.1-3-windows-x64-binaries\pgsql\bin to HeidiSQL install folder.
2. Dowload the latest (openssl-1.0.2q-x64_86-win64.zip) SSL DLLs (libeay32.dll and ssleay32.dll) from https://indy.fulgan.com/SSL/  and copy them to HeidiSQL install folder.

**Comment by ansgarbecker** [comment]
Ok, thanks for testing. I will try out if the latest `libeay32` and `ssleay32` also work with the older libpq.dll, so I can provide both without separating the dependencies.

**Comment by ansgarbecker** [comment]
Here's an overview for 32/64bit PG releases: https://www.postgresql.org/download/windows/

**Comment by ansgarbecker** [comment]
Here we go: I just removed libpq.dll v9.5, and added 2 newer ones along with new dependencies: v10.0.11 32+64bit and v.12.0.1 64bit.

I found that both new versions don't use libeay32.dll and ssleay32.dll any longer. Instead, I had to add libcrypto-1_1.dll and libssl-1_1.dll.

Would be nice if someone out there could test the new nightly built installer from here: https://www.heidisql.com/download.php#r5859 (should be compiled in some minutes)
